PureLayout/PureLayout/ALView+PureLayout.h

```diff
@@ -118,6 +120,20 @@ __PL_ASSUME_NONNULL_BEGIN
 /** Pins an edge of the view to a given edge of another view with an offset as a maximum or minimum. */
 - (NSLayoutConstraint *)autoPinEdge:(ALEdge)edge toEdge:(ALEdge)toEdge ofView:(ALView *)otherView withOffset:(CGFloat)offset relation:(NSLayoutRelation)relation;
 
+/** Pin a space between self and the other view [self]-space-[otherView], either on the horizontal or vertical axis. */
+- (NSLayoutConstraint *)autoPinSpace:(CGFloat)space followedByView:(ALView *)otherView onAxis:(ALAxis)axis;
+
+/** Pin a space between self and the other view [self]-space-[otherView], either on the horizontal or vertical axis with the space as a maximum or minimum.  */
+- (NSLayoutConstraint *)autoPinSpace:(CGFloat)space followedByView:(ALView *)otherView onAxis:(ALAxis)axis relation:(NSLayoutRelation)relation;
+
+/** Pin trailing to leading with a space H:[self]-space-[otherView] */
+- (NSLayoutConstraint *)autoPinHorizontalSpace:(CGFloat)space followedByView:(ALView *)otherView;
+
+/** Pin bottom to top with a space V:[self]-space-[otherView] */
+- (NSLayoutConstraint *)autoPinVerticalSpace:(CGFloat)space followedByView:(ALView *)otherView;
```

Comment:
Sounds to me you just prefer the VFL way of expressing this, why not just use VFL for these then? What you're calling a space constraint is the same concept as pinning edges. If that wasn't already exposed in the library I would agree with you but this is just different naming for the same thing. Having 2 sets of methods to do the same thing is not a good idea.